javaslang/src/test/java/javaslang/control/OptionTest.java

```diff
@@ -70,41 +70,41 @@ public void shouldCreateNothing() {
 
     @Test
     public void shouldWrapIfTrue() {
-        assertThat(Option.some(null)).isEqualTo(Option.when(true, () -> null));
-        assertThat(Option.some(null)).isEqualTo(Option.when(true, (Object) null));
+        assertThat(Option.when(true, () -> null)).isEqualTo(Option.some(null));
+        assertThat(Option.when(true, (Object) null)).isEqualTo(Option.some(null));
```

Comment:
in OptionTest the `actual` and `expected` values were confused in multiple assertions